### PR TITLE
[Test/CodeClean] new testcases for coverage

### PR DESCRIPTION
**Pull request by jaeyun-jung**
Add new cases to improve test coverage and fix some minor comment.

Signed-off-by: Jaeyun <jy1210.jung@samsung.com>

### CODE REVIEW

File: tests/nnstreamer_sink/unittest_sink.cc [hunk 1]
```diff
@@ -4149,6 +4182,35 @@ TEST (tensorStreamTest, subpluginV0Run)
   g_free (fw);
 }
 
+/**
+ * @brief Test for hw availability using passthrough custom filter without model.
```
Comment (gichan-jang) [inline tests/nnstreamer_sink/unittest_sink.cc:4186]:
```suggestion
 * @brief Test for hw availability with invalid custom string.
```

**Review by gichan-jang — APPROVED** [review]
LGTM

File: tests/nnstreamer_sink/unittest_sink.cc (hunk 1 above)
Comment (jaeyun-jung) [reply tests/nnstreamer_sink/unittest_sink.cc]: updated

**Review by anyj0527 — APPROVED** [review]
LGTM, Thanks!

**Review by myungjoo — APPROVED** [review]

**Pull request merged**